### DISCUSSION TITLE
Just missed you!

### DISCUSSION THREAD

**Discussion by terminator356** [muse-sequencer/muse#1225] [Team Posts]
Ha! A mere few hours or so after the release, I finally pushed
 a brand new audio automation graphs complete rework
 that I've been working on for 5 months (a wee detour from my other work).

That's probably a good thing I pushed after you, since my changes
 are huge and may have a few bugs.

I'll try to write up a thingy about it for the documentation site.
A few more tweaks to come with this one, snapping and a few minor things.

Nice to cross paths with you today.
Tim.

**Comment by terminator356** [comment]
P.S. My code name and 'change stamp' for this 5 months of work all along
right from the start has been...

"Bunny"

Happy Easter...

**Comment by spamatica** [comment]
Haha, yeah. Interesting coincidence but all for the best I think. I  see there was already some feedback on some of the functionality, I guess some of it will result in some changes.
I will check it out myself asap!

And it will hopefully mean that we can make a 4.2 or 5.0 release all the faster.

Regarding documentation, I guess you saw the threads about involving more people for various tasks.
One of them resulting in a rework of the wiki which I think is in a pull-request.
I only saw the pull-request arrive so I'm not sure exactly what it contains but I saw a bunch of html files which may mean that the guy rewrote it as static html pages.

I fear I might have been naive in suggesting they just go ahead with updating it.
I'm guessing it may be easier to view but if they really are static pages it could be a real maintenance burden. Unless the guy takes a long term commitment to keeping it up to date...

Oh well. I'll check it out in a bit, I thought I'd send a heads up if you were planning to add to the manual.
